.gitignore

```diff
@@ -0,0 +1,2 @@
+### IntelliJ
+.idea/
```

Comment:
Why?
A useful .gitignore could exclude compiles OCaml files: `*.cmo *.cmi *.cmx *.cmxz`.
Feel free to steal mine: https://github.com/dmbaturin/soupault/blob/master/.gitignore

Reply:
Will do, thanks! I'm new to all this.